packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -283,7 +283,9 @@ export class AtomicBreadbox implements InitializableComponent {
       .flat()
       .map(({value, facetId, field}) => ({
         facetId,
-        label: this.bindings.store.state.facets[facetId].label,
+        label: this.bindings.store.state.facets[facetId]
```

Comment:
Had to do it this way to avoid getting the error "Cannot read property of undefined" in the console.

Reply:
how about `this.bindings.store.state.facets[facetId]?.label` ?